packages/@aws-cdk/aws-lambda/lib/adot-layers.ts

```diff
@@ -0,0 +1,251 @@
+  /**
+   * @param version either "LATEST" or the version string of the layer version (such as "1.19.0")The version of the ADOT Lambda layer
+   */
+  protected constructor(private readonly type: AdotLambdaLayerType, private readonly version: string) {
+    super();
+  }
+
+  /**
+   * Returns the arn of the ADOT Lambda layer based on the given configurations (ADOT type,
+   * version, and the Lambda function's architecture).
+   * @internal
+   */
+  public _bind(fn: Function): AdotLambdaLayerBindConfig {
+    return {
+      arn: getLayerArn(fn.stack, this.type, this.version, fn.architecture.name),
+    };
+  }
+}
+
+/**
+ * The collection of versions of the ADOT Lambda Layer for Java SDK
+ */
+export class AdotLambdaLayerJavaSdkVersion extends AdotLambdaLayerVersion {
+  /**
+   * Will always use the latest layer version. New versions could
+   * introduce incompatible changes. Make sure to test new versions.
+   */
+  public static readonly LATEST = new AdotLambdaLayerJavaSdkVersion('1.19.0');
+
+  /**
+   * Version 1.19.0
+   */
+  public static readonly V1_19_0 = new AdotLambdaLayerJavaSdkVersion('1.19.0');
+
+  private constructor(version: string) {
+    super(AdotLambdaLayerType.JAVA_SDK, version);
+  }
+}
+
+/**
+ * The collection of versions of the ADOT Lambda Layer for Java auto-instrumentation
+ */
+export class AdotLambdaLayerJavaAutoInstrumentationVersion extends AdotLambdaLayerVersion {
+  /**
+   * Will always use the latest layer version. New versions could
+   * introduce incompatible changes. Make sure to test new versions.
+   */
+  public static readonly LATEST = new AdotLambdaLayerJavaAutoInstrumentationVersion('1.19.2');
+
+  /**
+   * Version 1.19.2
+   */
+  public static readonly V1_19_2 = new AdotLambdaLayerJavaAutoInstrumentationVersion('1.19.2');
+
+  private constructor(version: string) {
+    super(AdotLambdaLayerType.JAVA_AUTO_INSTRUMENTATION, version);
+  }
+}
+
+/**
+ * The collection of versions of the ADOT Lambda Layer for Python SDK
+ */
+export class AdotLambdaLayerPythonSdkVersion extends AdotLambdaLayerVersion {
+  /**
+   * Will always use the latest layer version. New versions could
+   * introduce incompatible changes. Make sure to test new versions.
+   */
+  public static readonly LATEST = new AdotLambdaLayerPythonSdkVersion('1.13.0');
+
+  /**
+   * Version 1.13.0
+   */
+  public static readonly V1_13_0 = new AdotLambdaLayerPythonSdkVersion('1.13.0');
+
+  private constructor(version: string) {
+    super(AdotLambdaLayerType.PYTHON_SDK, version);
+  }
+}
+
+/**
+ * The collection of versions of the ADOT Lambda Layer for JavaScript SDK
+ */
+export class AdotLambdaLayerJavaScriptSdkVersion extends AdotLambdaLayerVersion {
+  /**
+   * Will always use the latest layer version. New versions could
+   * introduce incompatible changes. Make sure to test new versions.
+   */
+  public static readonly LATEST = new AdotLambdaLayerJavaScriptSdkVersion('1.7.0');
```

Comment:
Is there a pattern here that requires the use of a `LATEST` tag? The lambda layers don't really currently push any reference to `latest` and this would be the first introduction of the term in relation to the ADOT lambda layers. I worry that it could be confusing to end-users.

Reply:
If I update the wording as below, would that make more sense?

```
Use the latest layer version available within this CDK version ...
```

The `LATEST` version would be helpful for those who want to model a ADOT layer update via CDK library update.

Reply:
I went ahead and updated the comment. Please see the latest revision.